packages/client/hmi-client/src/layout/controller.ts

```diff
@@ -11,18 +11,23 @@ export default class LayoutController {
 
 	constructor() {
 		this.worker = new Promise((resolve, reject) => {
-			const worker = new Worker(new URL('./worker.ts', import.meta.url), {
-				type: 'module'
-			});
-			worker.onmessage = (e: any) => {
-				if (e.data) {
-					this._isReady = true;
-					resolve(worker);
-				} else {
-					console.error('Pyodide Init Error');
-					reject();
-				}
-			};
+			try {
+				const worker = new Worker(new URL('./worker.ts', import.meta.url), {
+					type: 'module'
+				});
+				worker.onmessage = (e: any) => {
+					if (e.data) {
+						this._isReady = true;
+						resolve(worker);
+					} else {
+						console.error('Pyodide Init Error');
+						reject();
+					}
+				};
+			} catch (e) {
+				console.error('Worker Creation Error:', e);
+				reject(e);
+			}
```

Comment:
FYI we moved this file into `src/web-workers/layout` to put all the web workers together. So we will need to adjust this after the fact.